kuma/settings/common.py

```diff
@@ -528,6 +528,7 @@ def _get_locales():
 
 STATICFILES_DIRS = (
     path('kuma', 'static'),
+    path('kuma', 'javascript', 'dist'),
```

Comment:
This works for now. I'd also like to consider ``/assets/src`` and ``/assets/dist`` as we migrate away from ``django-pipeline``, and move stuff out of ``/kuma/static``.